roles/test-linux-binary/tasks/main.yml

```diff
@@ -60,7 +60,7 @@
   wait_for: >
     path={{workdir}}/output/heartbeat timeout=5
     search_regex='"scheme":"http"'
-  when: beat_name == "heartbeat`"
+  when: beat_name == "heartbeat"
```

Comment:
What did the tick here do?

Reply:
It must have been a typo. So this task was never being executed.